tests/test_views_login.py

```diff
@@ -667,3 +667,40 @@ def test_remeber_token_throttling(self):
                                'auth-password': 'secret',
                                'login_view-current_step': 'auth'})
         self.assertRedirects(response, reverse(settings.LOGIN_REDIRECT_URL), fetch_redirect_response=False)
+
+    @mock.patch('two_factor.gateways.fake.Fake')
+    @mock.patch('two_factor.views.core.signals.user_verified.send')
+    @override_settings(
+        TWO_FACTOR_SMS_GATEWAY='two_factor.gateways.fake.Fake',
+        TWO_FACTOR_CALL_GATEWAY='two_factor.gateways.fake.Fake',
+        TWO_FACTOR_REMEMBER_COOKIE_AGE=60 * 60,
+    )
+    def test_issue_397(self, mock_signal, fake):
```

Comment:
Maybe this should be called something like `test_phonedevice_with_remember_cookie`